frontend/src/app/private-route/private-route.tsx

```diff
@@ -0,0 +1,15 @@
+import { useLoginState } from "../profile/hooks";
+import React, { Component } from "react"
+import { Redirect } from "@reach/router";
+
+
+export const PrivateRoute = ({component: Component, location, lang, ...rest}: any) => {
+  const [loggedIn, _] = useLoginState(false);
+  const loginUrl = `${lang}/login`;
+
+  if (!loggedIn && !location.pathname.includes("login")) {
```

Comment:
Why this `location.pathname.includes("login")` check?

Reply:
Because if you are already in the "login" page we don't need to redirect... 
[location.path includes `/{lang}/` so I checked _if includes_ login, not if is equal...]

Reply:
The loging route won't be a PrivateRoute no? :) it would never get here

Reply:
Ah, That's right! 
I copied the [documentation](https://www.gatsbyjs.org/tutorial/authentication-tutorial/#controlling-private-routes) 😅, but in our case, we don't need it :)

Reply:
fixed :)